Sources/PrimerSDK/Classes/PCI/Checkout Components/PrimerRawCardDataTokenizationBuilder.swift

```diff
@@ -60,6 +60,14 @@ class PrimerRawCardDataTokenizationBuilder: PrimerRawDataTokenizationBuilderProt
         }
     }
     
+    
+    /// Indicates whether or not one of the co-badged networks is present in the merchant config
+    /// If it is present, the remote card validation service will be called during card validation
+    var isCoBadgedCardsEnabled: Bool {
+        let supportedCardNetworks: [CardNetwork] = .supportedCardNetworks
+        return supportedCardNetworks.contains{ CardNetwork.coBadgedNetworks.contains($0) }
```

Comment:
TIL something. This syntax is a brainteaser... but nice usage 🙃

Not knowing this predicate-based contains, I probably would have ended up on something like `!Set(supportedCardNetworks).intersection(Set(CardNetwork.coBadgedNetworks)).isEmpty` but isn't a whole lot easier to grok in the end...

Not suggesting a change btw

Reply:
This is similar to what Andre suggested. Yeah, sets aren't that widely used ... but doesn't mean we can't though. In fact we could make the settings field type `Set<CardNetwork>` too to avoid duplication and make this a cleaner comparison.

I'll try a few things and see how it looks.

> This syntax is a brainteaser...

This is good enough reason to change it for me, I already don't like this syntax 😅 I've used an extension on Arrays to add a `contains(oneOf: ...)` method previously, but it seems overkill for this situation.

Reply:
I changed the config to expect a `Set` and it works great with array literals 🎉